packages/wonder-blocks-popover/src/components/__docs__/popover-content.stories.js

```diff
@@ -0,0 +1,134 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Button from "@khanacademy/wonder-blocks-button";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+
+import type {StoryComponentType} from "@storybook/react";
+import {PopoverContent} from "@khanacademy/wonder-blocks-popover";
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+import PopoverContentArgtypes from "./popover-content.argtypes.js";
+
+export default {
+    title: "Popover/PopoverContent",
+    component: PopoverContent,
+    argTypes: PopoverContentArgtypes,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
```

Comment:
I haven't putting this `docs` section in the components I've been migrating. Is this something I should have been doing?

Reply:
ahh yeah, sorry that I totally missed that.

The thing is that `storyDescription` still works as of v6, but it will be deprecated in v7 in favor of `docs.description`. You can do that on your next stories, and we can modify the previous ones once we migrate to Storybook v7 (I suspect we'll be able to update at some point in Q4).